### PR TITLE
Provone

### PR DESCRIPTION
**Pull request by sanuann**
@satra - Please review.

### CODE REVIEW

**Comment by djarecka** [comment]
@sanuann - travis is failing because there is no `urllib.parse` in py2,  `from urlparse import urlparse` should work for py2.

**Comment by djarecka** [comment]
I haven't had time to fully debug the failing test `nidm/experiment/tests/test_experiment.py` (commented right now), but it looks like only importing `import nidm.core.provone` make the test fail. I prepared much simpler tests that show the error, maybe this will help:

```
from nidm.experiment import Project
from nidm.core import Constants


def test_pass():
    project = Project()
    project_PI = project.add_person(attributes={Constants.NIDM_FAMILY_NAME:"Keator", Constants.NIDM_GIVEN_NAME:"David"})
    project.add_qualified_association(person=project_PI,role=Constants.NIDM_PI)

    with open("test.ttl",'w') as f:
        f.write (project.serializeTurtle())

def test_fail():
    # adding import                                                                                                                                             
    import nidm.core.provone

    project = Project()
    project_PI = project.add_person(attributes={Constants.NIDM_FAMILY_NAME:"Keator", Constants.NIDM_GIVEN_NAME:"David"})
    project.add_qualified_association(person=project_PI,role=Constants.NIDM_PI)

    with open("test.ttl",'w') as f:
        f.write (project.serializeTurtle())
```

File: nidm/core/tests/test_provone.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    d1.add_namespace('wfms', 'http://www.wfms.org/registry/')
+    d1.add_namespace('rdfs', 'http://www.w3.org/2000/01/rdf-schema#')
+    d1.add_namespace('nowpeople', 'http://www.provbook.org/nownews/people/')
+    d1.add_namespace('xsd', 'http://www.w3.org/2001/XMLSchema#')
+    d1.add_namespace('owl', 'http://www.w3.org/2002/07/owl#')
+
+    return d1
+
+
+def test_ispartof(doc):
```
Comment (satra) [inline nidm/core/tests/test_provone.py]: these tests should not just test writing out a file, it should also compare or determine what was written to be correct. perhaps @djarecka can help @sanuann to improve these tests.

**Comment by satra** [comment]
i'm going to merge this into master, so we can start using them.

**Comment by djarecka** [comment]
@satra this PR comented an existing test because it was failig

**Pull request merged**